docs/building.md

```diff
@@ -38,13 +38,61 @@ ctest -L performance
    |    TEST\_INTEL\_OPTIMIZATION | OFF      | Option | Test intel optimization and vectorization |
    |    CHECK\_REGISTRY\_PRESSURE | OFF      | Option | Check the registry pressure for Kokkos CUDA kernels |
    |               BUILD\_TESTING | ON       | Option | Multi-testing enablement |
-   | PARTHENON\_DISABLE\_EXAMPLES | OFF      | Option | Toggle building of examples, if regression tests are on, drivers needed by the tests will still be built |   
+   | PARTHENON\_DISABLE\_EXAMPLES | OFF      | Option | Toggle building of examples, if regression tests are on, drivers needed by the tests will still be built |
    |   ENABLE\_INTEGRATION\_TESTS | ${BUILD\_TESTING} | Option | Enable integration tests |
    |    ENABLE\_REGRESSION\_TESTS | ${BUILD\_TESTING} | Option | Enable regression tests |
    |  REGRESSION\_GOLD\_STANDARD\_VER | #     | Int    | Version of current gold standard file used in regression tests. Default is set to latest version matching the source. |
    | REGRESSION\_GOLD\_STANDARD\_HASH | SHA512=... | String | Hash value of gold standard file to be downloaded. Used to ensure that the download is not corrupted. |
    | REGRESSION\_GOLD\_STANDARD\_SYNC | ON    | Option | Create `gold_standard` target to download gold standard files |
    |          ENABLE\_UNIT\_TESTS | ${BUILD\_TESTING} | Option | Enable unit tests |
    |               CODE\_COVERAGE | OFF      | Option | Builds with code coverage flags |
+   |       CMAKE\_INSTALL\_PREFIX | machine specific | String | Optional path for shared library installation |
 
 ### NB: CMake options prefixed with *PARTHENON\_* modify behavior.
+
+## Building a shared library
+
+An alternative to building Parthenon alongside a custom app (as in the examples)
+is to first build Parthenon separately as a shared library and then link to it
+in the app.
+
+To build Parthenon as a shared library, provide a *CMAKE\_INSTALL\_PREFIX* path
+to the desired install location to the Parthenon cmake call, then build and install
+(note that `--build` and `--install` required CMake 3.15 or greater)
+
+```bash
+cmake -DCMAKE_INSTALL_PREFIX="$your_install_dir" $parthenon_source_dir
```

Comment:
Put this is a separate heading about "building as a shared library", but the user needs to specify `-DBUILD_SHARED_LIBS=ON`, too.